<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformationNodeGUI</class>
 <widget class="QWidget" name="TransformationNodeGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Transformation Properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Translate</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="translate_x"/>
          </item>
          <item>
           <widget class="QLineEdit" name="translate_y"/>
          </item>
          <item>
           <widget class="QLineEdit" name="translate_z"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Rotate</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLineEdit" name="rotate_x"/>
          </item>
          <item>
           <widget class="QLineEdit" name="rotate_y"/>
          </item>
          <item>
           <widget class="QLineEdit" name="rotate_z"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Scale</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="scale_x"/>
          </item>
          <item>
           <widget class="QLineEdit" name="scale_y"/>
          </item>
          <item>
           <widget class="QLineEdit" name="scale_z"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>translate_x</sender>
   <signal>editingFinished()</signal>
   <receiver>TransformationNodeGUI</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>translate_y</sender>
   <signal>editingFinished()</signal>
   <receiver>TransformationNodeGUI</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>translate_z</sender>
   <signal>editingFinished()</signal>
   <receiver>TransformationNodeGUI</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>786</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>758</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotate_x</sender>
   <signal>editingFinished()</signal>
   <receiver>TransformationNodeGUI</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotate_y</sender>
   <signal>editingFinished()</signal>
   <receiver>TransformationNodeGUI</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotate_z</sender>
   <signal>editingFinished()</signal>
   <receiver>TransformationNodeGUI</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>869</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>835</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scale_x</sender>
   <signal>editingFinished()</signal>
   <receiver>TransformationNodeGUI</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scale_y</sender>
   <signal>editingFinished()</signal>
   <receiver>TransformationNodeGUI</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scale_z</sender>
   <signal>editingFinished()</signal>
   <receiver>TransformationNodeGUI</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>947</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>932</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Update()</slot>
 </slots>
</ui>
